src/components/shared/Entity/hooks/useEntityWorkflowHelpers.ts

```diff
@@ -131,35 +137,42 @@ function useEntityWorkflowHelpers() {
     const { generatePath } = useDetailsNavigator(entityDetailsBaseURL);
 
     const exit = useCallback(
-        (customRoute?: string) => {
+        (customRoute?: string, external?: boolean) => {
```

Comment:
Why are we passing both of these? I can only see them being passed together in the `HeaderLogs` component.

Reply:
There is at least one instance of a custom route being passed to this function that is within the context of the application. I would argue conflating them would be folly, given any route within the context of the application should be handled by the function returned by `useNavigate`.